packer/build_image.sh

```diff
@@ -190,31 +190,21 @@ check_deb_exists () {
     done
 }
 
-import_gpg_key () {
-  echo "Importing apt key ($APT_KEY)"
-  TMPREPO=$(mktemp -u -p /etc/apt/sources.list.d/ --suffix .list)
-  sudo curl -sSo $TMPREPO $REPO_FOR_INSTALL
-  sudo mkdir -p $APT_KEYS_DIR
-  sudo gpg --homedir /tmp --no-default-keyring --keyring $APT_KEYS_DIR/scylladb.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $APT_KEY
-  sudo apt-get update -y
-}
-
 if [ -z "$TARGET" ]; then
     echo "Missing --target parameter. Please specify target cloud (aws/gce/azure)"
     exit 1
 fi
 
 SSH_USERNAME=ubuntu
 
+SCYLLA_FULL_VERSION="$VERSION-$SCYLLA_RELEASE"
+SCYLLA_MACHINE_IMAGE_VERSION="$VERSION-$SCYLLA_MACHINE_IMAGE_RELEASE"
+
 if [ $LOCALDEB -eq 1 ]; then
     INSTALL_ARGS="$INSTALL_ARGS --localdeb"
```

Comment:
BTW, what about the option of `--localdeb` I'm not sure it will work now. Actually I'm not sure we are using it

Reply:
We are not using it for sure. it's for someone who builds locally and then needs an AMI
We can obliterate it as we have today BYO can do that instead, WDYT?

Reply:
Yes, I think BYO it is a good alternative.

Reply:
I will add another commit to remove it